lib/themes/dosomething/paraneue_dosomething/js/app.js

```diff
@@ -44,17 +44,15 @@ define("app", function (require) {
       Donate.init();
     }
 
-
-    var $mosaicGalleries = $body.find(".gallery.-mosaic");
+    var $revealGalleries = $body.find("ul[data-role='show_more']");
```

Comment:
You probs don't need the `ul` on there. Would make it more generic to apply to more things if it's just the `[data-role='show_more']` :)